### DISCUSSION TITLE
v20: update product websites -> notify about index

### DISCUSSION THREAD

**Discussion by f1-outsourcing** [OpenMage/magento-lts#3963] [Ideas]
I am not entirely sure when you get these notifications when to update the indexes. But when you modify websites of a product, it is required to update indexes, maybe add this to the notification? There is also no red index shown.

**Comment by pquerner** [comment]
Where did you modify the website? Inside a product (for example override an attribute) or what do you mean?

**Comment by f1-outsourcing** [comment]
catalog->products -> click a product -> websites -> select website ->save

this product is not show on the page

system -> index management 
all indexes are green

select all -> reindex

now the product is show on the page. 

When testing this, I noticed that when removing this product the change is immediately visible.

**Reply by pquerner** [reply]
Sorry, its still not clear what you are updating.

You are updating the product assignment to a website, yes? 

I've tested this on my system and it works as expected. After updating the assignment, all indexes become red - needing a reindex.

**Reply by f1-outsourcing** [reply]
Strange mine stay green. Maybe it is related to that I have disabled cache management?

**Reply by pquerner** [reply]
No, cache shouldnt interfere here.
